docs/images_and_data/uml_sequence.txt

```diff
@@ -0,0 +1,46 @@
+activate Csv2TexController
```

Comment:
I omitted the boring stuff, CsvParsingUtil, CsvValidator, and anything except the "happy flow" (so, showing no exception handling in this diagram).

https://github.com/SergelsOrg/csv2tex/tree/main/src/main/java/org/example/csv2tex/csv